src/main/java/com/dataloom/graph/core/LoomGraph.java

```diff
@@ -2,56 +2,64 @@
 
 import java.util.UUID;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import com.dataloom.data.EntityKey;
 import com.dataloom.graph.core.objects.EdgeKey;
+import com.dataloom.graph.core.objects.EdgeLabel;
+import com.dataloom.graph.core.objects.EdgeSelection;
 import com.dataloom.graph.core.objects.GraphWrappedEdgeKey;
 import com.dataloom.graph.core.objects.GraphWrappedEntityKey;
 import com.dataloom.graph.core.objects.GraphWrappedVertexId;
 import com.dataloom.graph.core.objects.LoomEdge;
 import com.dataloom.graph.core.objects.LoomVertex;
 import com.dataloom.graph.core.objects.VertexLabel;
 import com.dataloom.hazelcast.HazelcastMap;
+import com.datastax.driver.core.utils.UUIDs;
 import com.hazelcast.core.HazelcastInstance;
 import com.hazelcast.core.IMap;
 
 public class LoomGraph implements LoomGraphApi {
-    
-    private static UUID DEFAULT_GRAPH_ID = new UUID(0, 0);
-    private UUID graphId;
-    
+
+    private static final Logger                           logger           = LoggerFactory.getLogger( LoomGraph.class );
+    private static UUID                                   DEFAULT_GRAPH_ID = new UUID( 0, 0 );
+    private UUID                                          graphId;
+
     private static IMap<GraphWrappedVertexId, LoomVertex> vertices;
-    private static IMap<GraphWrappedEntityKey, UUID> verticesLookup;
-    
-    private static IMap<GraphWrappedEdgeKey, LoomEdge> edges;
-    private static GraphQueryService gqs;
-    
-    public static void init( HazelcastInstance hazelcastInstance, GraphQueryService gqs ){
+    private static IMap<GraphWrappedEntityKey, UUID>      verticesLookup;
+    private static IMap<GraphWrappedEdgeKey, LoomEdge>    edgeLookup;
```

Comment:
this is not necessary. The `verticesLookup` was needed to lookup vertex ID by entity key. Don't think there is a similar situation for edges.